### PR TITLE
Research pdf can be uploaded to the app (API)

### PR DESCRIPTION
**Pull request by julieahlquist**
### User Story

```
As a Research Group,
In order to allow readers to read the complete research,
I want to have be able to upload PDF documents so readers can download.
```
### Feature description
Link to this [feature on Pivotal Tracker](https://www.pivotaltracker.com/story/show/168187877)

### Description of PR
Adds ability to attach a pdf file with the original research paper. 

### What did we learn?
We learned about AWS and Base64

### CODE REVIEW

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -1 +1,2 @@
...
+jl+wLK9x+X7bsdzyEHsg2gLwarkRgazPb304CeUFeYXYtz1EM2PZ0CqGk4UKLtZGSaC/fN2INjesPgOh84vgFeq+TqPa1lf+7kTHW/h39lz31tNARyHqt+/lraMQfWA1hTXdElKUp9ByaRTcFuYk6aXONucILCP5lfVftZ+YkCkMjtUOx8qFRT/reQ4EV3pjbA66tEav4vMMs2zE/P0Nk6LTRcaQO4JcrnaPbCY3s0h3X3dvKNkl+M9dIR8RCYQc/Qf3RX6Cx/ZwzpyURPSuYcNZl/nwvEEAW0OKgL//CEIr60Aq6MPx/m1jcz4spibDRSbENq0KMTVivyhJaDPq8yWyIMtxURDpiONlnWCl8McKPYe6Gqqz5gk6sGnxQaD7p4XoV88tDmwskEfET0LA01N0TIJIdy9K30MoWzWWt0gI/n5owov5vC7Ond72Yal7LxJEp31BBw7ZyNhlaKeXpbP7S9iiRIe5JZsbQ3BobWFyZtQMQFWqlmzU1HaOBHruvx1sobSzMdNT2VlK4x3eAvVgv0QQcgVp7MbPK22hguCGdgI2bKi+F6+Y084uKUsJ1jB0Ok5klTle2OdjaBPeTlDXbAa9iMyhNsmUTalxiuqQGBj4ArnK4yNDS18hPHwVtpz5mmwNTy5q7DKDMq1Vr61MX0ViGxsJnbFsdaV9Y0m71MjxV45dp7Vg9LjoDcpT640MDh9Kzd2F9I9rhd8t--0L/joGliot1jolOl--lkIEvWWxQxfzhqMh24jlcg==
```
Comment (oliverochman) [inline config/credentials.yml.enc]: this does not look right

Comment (oliverochman) [reply config/credentials.yml.enc]: can u try to see the credentials

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: config/credentials.yml.enc (hunk 1 above)
Comment (julieahlquist) [reply config/credentials.yml.enc]: fixed

**Review by oliverochman — APPROVED** [review]
Good job

**Pull request merged**